avocado/utils/ssh.py

```diff
@@ -20,6 +20,10 @@ class Session:
     """
     Represents an SSH session to a remote system, for the purpose of
     executing commands remotely.
+
+    :class:`Session` is also a context manager. On entering the context
+    it tries to establish the connection, therefore on exiting that
+    connection is closed. Usage example:
```

Comment:
Looks like you're missing the example here.

Reply:
Good catch, I thought initialize on place an example here. Later I figured out it has the `examples/apis/utils/ssh.py` that I included in the module's doc. So I should remove this `Usage example:` here.